python/cudf/cudf/core/series.py

```diff
@@ -5009,12 +5009,73 @@ def _get_dt_field(self, field):
         )
 
     def ceil(self, field):
+        """
+        Perform ceil operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        field : str
+            One of ["D", "H", "T", "S", "L", "U", "N"]
+            See `frequency aliases <https://pandas.pydata.org/docs/\
+                user_guide/timeseries.html#timeseries-offset-aliases>`_
+            for more details on these aliases.
```

Comment:
I see the docstrings for [Series.dt.ceil](https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.ceil.html#pandas.Series.dt.ceil) say:

> Must be a fixed frequency like ‘S’ (second) not ‘ME’ (month end). See [frequency aliases](https://pandas.pydata.org/docs/user_guide/timeseries.html#timeseries-offset-aliases) for a list of possible freq values.

That requirement of a "fixed frequency" could be clarified in this docstring.